example/.travis.yml

```diff
@@ -74,13 +74,13 @@ env:
     # valid path to a YAML file relative to ORCA itself, e.g.,
     # ../example/tests/packages.yml.
     # @see https://github.com/acquia/orca/blob/develop/config/packages.yml
-    # - ORCA_PACKAGES_CONFIG: ../example/tests/packages.yml
+    - ORCA_PACKAGES_CONFIG=../example/tests/packages.yml
```

Comment:
This should stay commented out. (Remember that this file is supposed to be copied into new projects as a template, and lines like this wouldn't be appropriate by default--they serve only as an example.)